src/jit/assertionprop.cpp

```diff
@@ -2963,12 +2963,24 @@ AssertionIndex Compiler::optLocalAssertionIsEqualOrNotEqual(
     return NO_ASSERTION_INDEX;
 }
 
-/*****************************************************************************
- *
- *  Given a set of "assertions" to search for, find an assertion that is either
- *  "op1" == "op2" or "op1" != "op2." Does a value number based comparison.
- *
- */
+//------------------------------------------------------------------------
+// optGlobalAssertionIsEqualOrNotEqual: Look for an assertion in the specified
+//        set that is one of op1 == op1, op1 != op2, *op1 == op2, *op1 != op2,
```

Comment:
The code below only allows for  `*op1 == op2`, not `*op1 != op2`

Reply:
Thanks. Meant just to handle the equality case.